x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/rest/action/SecurityBaseRestHandlerTests.java

```diff
@@ -28,10 +28,11 @@
 
 public class SecurityBaseRestHandlerTests extends ESTestCase {
 
-    public void testSecurityBaseRestHandlerChecksLicenseState() throws Exception {
+    public void testSecurityBaseRestHandlerChecksLicenseStateBeforePreparingRequest() throws Exception {
```

Comment:
Nit:
```suggestion
    public void testSecurityBaseRestHandlerChecksFeatureAvailableBeforePreparingRequest() throws Exception {
```

This test actually does not check license at all. I guess it was license related before security got basic licensed. But it's no longer the case. We can probably remove the `when(...).thenReturn(...)` setup for the licenseState mock as well.